package.json

```diff
@@ -28,7 +28,7 @@
         "pwa:start": "lerna run start --scope @padloc/pwa",
         "server:start": "lerna run start --scope @padloc/server --stream",
         "server:start-dry": "lerna run start-dry --stream --scope @padloc/server",
-        "electron:build": "cd packages/electron && npm run build && cd ../..",
+        "electron:build": "lerna run build --scope @padloc/electron",
```

Comment:
This just makes it consistent and links everything as expected.

Reply:
I *think* there was a reason why I did it this way, but I can't remember it now. Since builds seem to be unaffected I assume this change is ok so let's keep it. I just hope this isn't one of those Chesterton's Fence moments 😄